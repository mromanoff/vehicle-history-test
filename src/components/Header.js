import React, { Component } from 'react';
import Logo from './Logo';

class Header extends Component {
  render() {
    return (
      <header role="banner" className="header">
        <h1 className="header__logo">
          <Logo/>
          <span>VehicleHistory code test</span>
        </h1>
      </header>
    );
  }
}

export default Header;
