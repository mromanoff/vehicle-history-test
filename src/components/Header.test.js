import React from 'react';
import Header from './Header';

it('Header renders correctly', () => {
  const wrapper = shallow(<Header />);
  expect(wrapper).toMatchSnapshot();
});
