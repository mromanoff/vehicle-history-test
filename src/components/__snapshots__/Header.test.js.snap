// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header renders correctly 1`] = `
<header
  className="header"
  role="banner"
>
  <h1
    className="header__logo"
  >
    <Logo />
    <span>
      VehicleHistory code test
    </span>
  </h1>
</header>
`;
